fdbt-aws/cloudformation/service/eu-west-2/cognito.yml

```diff
@@ -40,7 +40,7 @@ Resources:
     Type: AWS::Logs::LogGroup
     Properties:
       LogGroupName: !Sub /aws/lambda/cognito-email-trigger-${Stage}
-      RetentionInDays: 60
+      RetentionInDays: 180
```

Comment:
according to Google, 6 months is 182.5 days. Should we go to 183?

Reply:
Ah right, so in the AWS console, 1 month = 20 days, so 6 months = 180 days. This means that only 180 days displays as 6 months in the console. In terms of satisfying the Accenture logging requirement, I think this will do?